docs/source/user_guide/installation/gnu-linux-mac.rst

```diff
@@ -101,12 +100,15 @@ Users can install ``jax`` and ``jaxlib`` to use the Jax solver.
 
 The ``pip install "pybamm[jax]"`` command automatically downloads and installs ``pybamm`` and the compatible versions of ``jax`` and ``jaxlib`` on your system.
 
+PyBaMM's full `conda-forge distribution <index.html#installation>`_ (``pybamm``) includes ``jax`` and ``jaxlib`` by default.
+
 .. _optional-iree-mlir-support:
 
 Optional - IREE / MLIR support
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Users can install ``iree`` (for MLIR just-in-time compilation) to use for main expression evaluation in the IDAKLU solver. Requires ``jax``.
+At the moment, PyBaMM's conda-forge distribution does not include the ``IDAKLUSolver``.
```

Comment:
I think this should be mentioned somewhere better – either in a dedicated "Difference between PyBaMM's `pip` and `conda` distributions" section and/or somewhere earlier on the page where we highlight the commands to install PyBaMM, since we can't expect that everyone will scroll down to this part of the page after copying the relevant commands. It should also be under an admonition for emphasis.

Reply:
I am going to try to wrap up the IDAKLU separation before 25.1, that was the target for big changes. Then this won't be an issue